code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -47,11 +47,10 @@
 
 /obj/structure/closet/secure_closet/freezer/kitchen/maintenance/PopulateContents()
 	..()
-	for(var/i = 0, i < 5, i++)
+	for(var/i in 1 to 5)
 		new /obj/item/reagent_containers/food/condiment/milk(src)
-	for(var/i = 0, i < 5, i++)
```

Comment:
this will probably mess with ordering?

Reply:
Ordering? Like what is on top of eachother? 

Is that something to be preserved?

Reply:
It's possible? Maybe compare ingame. I don't know if it even does

Reply:
1b08f2a